packages/node/src/integrations/onuncaughtexception.ts

```diff
@@ -7,6 +7,28 @@ import { logAndExitProcess } from './utils/errorhandling';
 
 type OnFatalErrorHandler = (firstError: Error, secondError?: Error) => void;
 
+interface OnUncaughtExceptionOptions {
+  // TODO(v8): Evaluate whether we should switch the default behaviour here.
+  // Also, we can evaluate using https://nodejs.org/api/process.html#event-uncaughtexceptionmonitor per default, and
+  // falling back to current behaviour when that's not available.
+  /**
+   * Whether the SDK should mimic native behaviour when a global error occurs. Default: `true`
+   * - `false`: The SDK will exit the process on all uncaught errors.
+   * - `true`: The SDK will only exit the process when there are no other 'uncaughtException' handlers attached.
+   */
+  exitEvenWhenOtherOnUncaughtExceptionHandlersAreRegistered: boolean;
```

Comment:
l: Hmm on the one hand this name is very verbose and I like that. on the other hand, we _might_ be able to shorten it a little without loosing much:

```suggestion
  exitEvenIfOtherHandlersAreRegistered: boolean;
```

If think we can get away just with "handlers" because we're in the scope of the `OnUncaughtException` integration.

WDYT?

Reply:
Sounds like a good compromise to me! 👍

Reply:
Yup that reads well!